### PR TITLE
EPMRPP 80384 || notification. Add AND/OR  attribute options

### PR DESCRIPTION
**Pull request by Bam6ycha**

### CODE REVIEW

**Review by chivekrodis — APPROVED** [review]

File: app/src/pages/common/settingsPage/notificationsTab/modals/addEditNotificationCaseModal/addEditNotificationCaseModal.jsx [hunk 1]
```diff
@@ -51,15 +51,8 @@ const messages = defineMessages({
...
 @reduxForm({
-  form: 'notificationCaseForm',
-  validate: (
-    { ruleName, recipients, informOwner, launchNames, attributes },
-    { data: { notificationCase, cases } },
-  ) => ({
-    ruleName: commonValidators.createRuleNameValidator(
-      cases.map((item) => ({ name: item.ruleName, ...item })),
-      notificationCase && notificationCase.id,
-    )(ruleName),
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/notificationsTab/modals/addEditNotificationCaseModal/addEditNotificationCaseModal.jsx:62]: Why did you remove the `ruleName` field validation?
It is required in the 5.8 API.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/common/settingsPage/notificationsTab/modals/addEditNotificationCaseModal/addEditNotificationCaseModal.jsx (hunk 1 above)
Comment (Bam6ycha) [reply app/src/pages/common/settingsPage/notificationsTab/modals/addEditNotificationCaseModal/addEditNotificationCaseModal.jsx:62]: That is great point.
I just cherry-pick commit from 5.7.4 and this form have not ruleName field validation.
I will return this one.

**Review by AmsterGet — APPROVED** [review]
